internal/handlers/handler_test.go

```diff
@@ -422,6 +422,12 @@ func TestReadOnlyHandlers(t *testing.T) {
 	}
 
 	hostname, err := os.Hostname()
+
+	buildEnvironment := types.MustMakeDocument()
+	for k, v := range version.Get().BuildEnvironment {
+		buildEnvironment.Set(k, v)
+	}
```

Comment:
Can we test specific values instead of copying a part of implementation?

Reply:
`BuildInfo.Settings` is empty when `go test` is run, so `buildInfo.buildEnvironment` field is actually empty when the project is tested. What do we do?

Reply:
So that code actually does nothing then :) Let's remove it